stylesheets/_component.tables.scss

```diff
@@ -158,6 +158,7 @@
 }
 
 .table-action {
+    border-bottom: none !important;
```

Comment:
Could we work around using `!important` here? I'm guessing the aim is to remove the bottom border recently added by `.table [href]` - which seems a little too unspecific? Could we address the issue there perhaps? Either change it to use a more specific selector or chain `.table-action` here to bump specificity? Perhaps it's time to remove the right-hand separators on `.table-action`? I'd rather we kept most if not all standard links underlined. Alternatively, you could remove the `table-action` class from the links in CMS (which is what I did in CXM).

Reply:
> Perhaps it's time to remove the right-hand separators on .table-action

I'd agree with that, the underlining of the links should be enough to visually separate actions now